source/_integrations/azure_event_hub.markdown

```diff
@@ -32,6 +36,19 @@ azure_event_hub:
   event_hub_instance_name: EVENT_HUB_INSTANCE_NAME
   event_hub_sas_policy: SAS_POLICY_NAME
   event_hub_sas_key: SAS_KEY
+  send_interval: 5
+  max_delay: 30
+  filter:
+    include_domains:
+    - homeassistant
+    - light
+    - media_player
+
+# Alternative config
+azure_event_hub:
+  event_hub_connection_string: CONNECTION_STRING
+  send_interval: 5
+  max_delay: 30
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.